spec/index.bs

```diff
@@ -13540,7 +13539,6 @@ specified points.
 
     - "update the rendering or user interface of that `Document`"
     - "update the rendering of that dedicated worker"
-    - "update the rendering of that service worker"
```

Comment:
I see "update the rendering of that service worker" doesn't appear in the HTML spec. I guess you can't render from a ServiceWorker or SharedWorker? But I don't see anything in the HTML spec saying you can't transferControlToOffscreen+postMessage to a ServiceWorker or SharedWorker. Maybe it's just underspecified in the HTML spec. Do you know if Chrome allows it? (If it does, we better propose an HTML spec change.)

Reply:
I was able to call `const canvas = new OffscreenCanvas(256, 256); const context = canvas.getContext("webgpu");` from a Service Worker with a Chromium build patched with https://chromium-review.googlesource.com/c/chromium/src/+/5190750

So I think it's "only" underspecified or we're not only looking at the right spec.

Reply:
@kainino0x I've filed https://github.com/whatwg/html/issues/10112. Shall I update the PR with a link to this issue?
Something like  this?

```
Issue: Use "update the rendering of that worker" instead. See [whatwg/html#10112](https://github.com/whatwg/html/issues/10112)
```

Reply:
Done